src/test/inputs/test_config/config_bundle_test.ini

```diff
@@ -74,6 +74,7 @@ skirt_distance = 6
 skirt_height = 1
 skirts = 1
 small_perimeter_speed = 15
+small_perimeter_length = 20

```

Comment:
Maybe make this a bit larger by default?  20 mm would translate to a hole bit under 6.4 mm diameter.  I would suggest a default of 32, for a hole about 1 cm diameter.